core/src/main/java/org/apache/accumulo/core/fate/Fate.java

```diff
@@ -476,10 +477,18 @@ public Fate(T environment, FateStore<T> store, boolean runDeadResCleaner,
     this.workFinder.start();
   }
 
+  public Duration getInitialDelay() {
```

Comment:
Is the plan to allow overriding these methods? If not maybe they could be public and statically defined constants in Fate? Even if we want to keep the methods to allow overriding they could delegate to the statically defined constants (if you keep the methods they could be private). This way we are not creating new objects each time the method is called.

Reply:
Thanks for the suggestions. We shouldn't allow overriding for initial delay, so I removed that. `getDeadResCleanupDelay` is already overridden in `FastFate` and I also plan to override `getPoolWatcherDelay` in `FastFate` as well in an upcoming PR, so left those methods but no longer create a new Duration each time.